__docs__/wonder-blocks-accordion/accordion.stories.tsx

```diff
@@ -0,0 +1,291 @@
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+import type {Meta, StoryObj} from "@storybook/react";
+
+import {
+    Accordion,
+    AccordionSection,
+} from "@khanacademy/wonder-blocks-accordion";
+import {View} from "@khanacademy/wonder-blocks-core";
+import {Strut} from "@khanacademy/wonder-blocks-layout";
+import {tokens} from "@khanacademy/wonder-blocks-theming";
+import {LabelLarge} from "@khanacademy/wonder-blocks-typography";
+
+import ComponentInfo from "../../.storybook/components/component-info";
+import packageConfig from "../../packages/wonder-blocks-icon/package.json";
+
+import AccordionArgtypes from "./accordion.argtypes";
+
+export default {
+    title: "Accordion / Accordion",
+    component: Accordion,
+    parameters: {
+        componentSubtitle: (
+            <ComponentInfo
+                name={packageConfig.name}
+                version={packageConfig.version}
+            />
+        ),
+    },
+    argTypes: AccordionArgtypes,
+} as Meta<typeof Accordion>;
+
+type StoryComponentType = StoryObj<typeof Accordion>;
+
+const exampleSections = [
+    <AccordionSection header="First section">
+        This is the information present in the first section
+    </AccordionSection>,
+    <AccordionSection header="Second section">
+        This is the information present in the second section
+    </AccordionSection>,
+    <AccordionSection header="Third section">
+        This is the information present in the third section
+    </AccordionSection>,
+];
+
+/**
+ * By default, an accordion has a caret at the end of the header block and
+ * rounded corners.
+ */
+export const Default: StoryComponentType = {
+    args: {
+        children: exampleSections,
+        caretPosition: "end",
+        cornerKind: "rounded",
+        allowMultipleExpanded: true,
+    },
+};
+
+/**
+ * An accordion allows multiple sections to be expanded at the same time
+ * by default. However, if `allowMultipleExpanded` is set to `false`, only
+ * one section can be expanded at a time.
+ */
+export const AllowMultipleExpanded = {
```

Comment:
**suggestion:** It's probably better to skip this snapshot as well.... I was checking in chromatic and it doesn't seem to useful for visual regression testing.